Add `Lazer Grand Arena: Registrations now open!` wiki article and news post (#11599)
ppy/osu-wiki: news/2024/2024-06-18-lazer-grand-arena-registrations-now-open.md

---
layout: post
title: "Lazer Grand Arena: Registrations now open!"
date: 2024-06-18 10:00:00 +0000
---

> While it's great to see countries rallying together and forming teams to compete, sometimes we have to wonder what players could truly do if they didn't have to pick around their teammates and truly shine on their own, and with the recent improvements made to osu!'s future client, we feel like it's finally time to bring to you...

Reviewer: double "truly' sounds kinda weird

Author: tried a fix on 0e3d500

![](/wiki/Tournaments/LGA/2024/img/lga2024-banner.jpg)

THE LAZER GRAND ARENA!

> <div align="center">

Reviewer 2: fixes smol spacing

![image](https://github.com/ppy/osu-wiki/assets/36758269/59a90911-0e77-49ff-8947-c7c9e258b0fe)

```suggestion
<div align="center" class="osu-md__paragraph">
```

    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4?v=0" type="video/mp4" preload="none">
    </video>
</div>

> This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**

Reviewer: ```suggestion
This brand new event will pit the world's 32 best players against each other in a 1v1 played on **osu!(lazer)!**
```

> This brand new event will pit the world's 32 best players against eachother in a 1v1 played on **osu!(lazer)!**

Reviewer 2: ```suggestion
This brand-new event will pit the world's 32 best players against each other in 1v1 matches played on **osu!(lazer)!**
```

> The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle eachother using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!

Reviewer: ```suggestion
The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle each other using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

Author: applied similar reviews from wala (ty for the quick review though)

> The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full featured artist lineup, the players will now battle eachother using **Cumulative Score Difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!

Reviewer 2: ```suggestion
The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full Featured Artist lineup, the players will now battle each other using **cumulative score difference**! This means that if you win by 20k, you are up by 20k for the rest of the match. After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |

> | Registration phase | June 18 (10:00 UTC) to July 01 (23:59 UTC) |
> | Qualifier and tournament format showcase | July 06 (14:00 UTC) |

Reviewer 2: imo

```suggestion
| Registration phase | June 18 (10:00 UTC) to July 1 (23:59 UTC) |
| Qualifier and tournament format showcase | July 6 (14:00 UTC) |
```

---
layout: post
title: "Lazer Grand Arena: Registrations now open!"
date: 2024-06-18 10:00:00 +0000
---

While it's great to see countries rallying together and forming teams to compete, sometimes we have to wonder what players could do if they didn't have to pick around their teammates and could shine on their own, and with the recent improvements made to osu!'s future client, we feel like it's finally time to bring to you...

![](/wiki/Tournaments/LGA/2024/img/lga2024-banner.jpg)

THE LAZER GRAND ARENA!

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2024-06-18-lga-announcement.mp4" type="video/mp4" preload="none">
    </video>
</div>

This brand-new event will pit the world's 32 best players against each other in 1v1 matches played on **osu!(lazer)!**

The Grand Arena is going to be home to many experiments to the familiar tournament format. Featuring a full Featured Artist lineup, the players will now battle each other using **cumulative score difference**! This means that if you win by 20k, you are up by 20k for the rest of the match.

> After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to do some try out some new exciting ideas for players and viewers alike!

Reviewer 3: ```suggestion
After being drafted, all 5 maps will end up played, and the score difference at the end determines who wins the match. This allows us to try out some new exciting ideas for players and viewers alike!
```
Seems to be a small typo?
